Source/WebCore/platform/graphics/cocoa/MediaPlayerPrivateWebM.mm

```diff
@@ -978,7 +978,8 @@ - (void)removeAllVideoTargets;
 {
     if (needsFlush == NeedsFlush::Yes)
         flushTrack(trackId);
-    if (trackBuffer.reenqueueMediaForTime(time, timeFudgeFactor()))
+    bool isEnded = m_loadFinished && !m_pendingAppends;
```

Comment:
I think this won't be sufficient for webm; `Ref<GenericPromise> MediaPlayerPrivateWebM::seekTo(const MediaTime& time)` will not complete the seek if it's not within the buffered range, and the duration won't be.

If we want to handle this case, needs to be done there.